app/views/provider_interface/offer/conditions/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_with model: form_object, url: provider_interface_application_choice_offer_conditions_path(application_choice), method: form_method do |f| %>
+<%= form_with model: form_object, url: provider_interface_application_choice_offer_conditions_path(application_choice), method: form_method, local: false do |f| %>
```

Comment:
Submitting the form using AJAX has accessibility implications. We're going to redo this without.

Reply:
More details on this:
1. The idea of this feature is to speed up the data entry process in this form. Having to go back to the server for each button press results in no gain. (All it really achieves is no page reload).
2. We don't communicate that the form is loading very well (if at all), which is not accessible